lib/pharos/host/configurer.rb

```diff
@@ -13,7 +15,7 @@ def self.load_configurers
 
       # @return [Array]
       def self.configurers
-        @configurers ||= []
+        @configurers ||= Set.new
```

Comment:
Unrelated, but I don't see why it should iterate through the same class multiple times when searching for a match.